tests/github_actions/gha/.github/workflows/multiple_on_descendants.yaml

```diff
@@ -0,0 +1,27 @@
+name: Handle branches
+on:
+  pull_request:
+    types: [ opened, synchronize, labeled, unlabeled ]
+    workflow_dispatch:
```

Comment:
```suggestion
  pull_request:
    types: [ opened, synchronize, labeled, unlabeled ]
  workflow_dispatch:
```

Reply:
@nimrodkor this is a different structure than the one previously suggested. With this structure, the definition context will not contain the workflow_dispatch. It is not related to the fact we have multiple blocks under "on", since removing the "pull_request" also does not keep the workflow_dispatch. It's probably related to the fact the workflow_dispatch is empty. Let me know if the expectation is to see the workflow_dispatch in the final definition_context (even in case it's empty).

Reply:
This is the correct structure, so highly recommend changing the test file. 
Regarding the test result - I don't mind whether we actually have a "workflow_dispatch" node here, but I do want consistent behavior :). If it is consistent, let's leave it as is

Reply:
cool, changed the file and adjusted the test.